src/Helpers/Session/Donation.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace Give\Helper\Session\Donation;
+
+use function Give\Helper\Session\getSession;
+use function give_get_donation_id_by_key as getDonationIdByPurchaseKey;
+
+/**
+ * Get donation is from donor session.
```

Comment:
ID, not "is" I'm assuming

Reply:
https://github.com/impress-org/givewp/compare/1a732f01cf52...04064e2ec6fd